sage-ios/SAGE/SAGE/BrowseMentorsViewController.swift

```diff
@@ -108,6 +109,7 @@ class BrowseMentorsViewController: UITableViewController {
             self.alphabetizeAndLoad(mentorArray)
             
             }) { (errorMessage) -> Void in
+                self.showNoContentView()
```

Comment:
we dont wanna put this in the failure block because what if someone refreshes and it fails? we wanna at least keep the current data. and if the failure happens on the first try the error message will sow up so its ok if theres no "no content" message